src/Parser/Monk/Mistweaver/Modules/Talents/RisingMist.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { STATISTIC_ORDER } from 'Main/StatisticBox';
+import ExpandableStatisticBox from 'Main/ExpandableStatisticBox';
+import { formatNumber } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import HotTracker from '../Core/HotTracker';
+
+const debug = false;
+
+const RISING_MIST_EXTENSION = 2000;
+
+class RisingMist extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    hotTracker: HotTracker,
+  };
+
+  risingMistCount = 0;
+  risingMists = [];
+
+  efsExtended = 0; // tracks how many flourishes extended Wild Growth
+
+  remCount = 0;
+  efCount = 0;
+  evmCount = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.RISING_MIST.id);
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.RISING_SUN_KICK.id !== spellId) {
+      return;
+    }
+
+    this.risingMistCount += 1;
+    debug && console.log(`risingMist cast #: ${this.risingMistCount}`);
+
+    const newRisingMist = {
+      name: `risingMist #${this.risingMistCount}`,
```

Comment:
Probably better to have this in english instead of camelCase right? "Rising Mist #11" for example?